templates/desktop/developers.html

```diff
@@ -107,16 +107,17 @@
     </div>
 </section>
 
-<section class="row row-for-developers">
-    <div class="strip-inner-wrapper">
-        <div class="twelve-col">
-            <div class="seven-col">
-                <h3>Snaps are the new apps</h3>
-                <p>Ubuntu {{ lts_release_full }} introduces snaps: a new application packaging format.</p>
-                <p>Snaps are Ubuntu applications that are packaged alongside all their dependencies, making them much more robust. And they couldn’t be easier to write. You can use the language of your choice, be it Python, Go, C, C++, Node JS or even .NET, then package your snap from source using Snapcraft, a new open source tool. Which means practically no learning curve. </p>
-                <p>Snaps let you deliver updates unmodified, at your own pace. So whether you’re building for mobile, the desktop or the Internet of Things, there’s no faster, easier or more dependable way of getting your code to market.</p>
-                <p><a class="external" href="http://snapcraft.io">To find out more about snaps</a></p>
-            </div>
+<section class="row row-for-developers strip-light">
+    <div class="strip-inner-wrapper equal-height">
+        <div class="six-col equal-height__item">
+            <h2>Snaps are the new apps</h2>
+            <p>Ubuntu {{ lts_release_full }} introduces snaps: a new application packaging format.</p>
+            <p>Snaps are Ubuntu applications that are packaged alongside all their dependencies, making them much more robust. And they couldn’t be easier to write. You can use the language of your choice, be it Python, Go, C, C++, Node JS or even .NET, then package your snap from source using Snapcraft, a new open source tool. Which means practically no learning curve. </p>
+            <p>Snaps let you deliver updates unmodified, at your own pace. So whether you’re building for mobile, the desktop or the Internet of Things, there’s no faster, easier or more dependable way of getting your code to market.</p>
+            <p><a class="external" href="https://developer.ubuntu.com">To find out more about snaps</a></p>
```

Comment:
this should be 'Find out more about snaps' [external link: http://snapcraft.io] based on the copy doc